packages/frontend/src/components/UserCompletionModal/UserCompletionModal.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { waitFor } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { renderWithProviders } from '../../testing/testUtils';
+import UserCompletionModal from './UserCompletionModal';
+
+describe('UserCompletionModal', () => {
+    it("should not render anything if user's setup is complete", async () => {
+        const { container } = renderWithProviders(<UserCompletionModal />);
+
+        await waitFor(() => expect(container).toBeEmptyDOMElement());
+    });
```

Comment:
by default `renderWithProviders` mocks user's data as a regular logged in user with `isSetupComplete = true` which makes `<UserCompletionModal />` not to render